### DISCUSSION TITLE
Are we data protection layer yet?

### DISCUSSION THREAD

**Discussion by kocoten1992** [stratis-storage/project#534] [Q&A] [answered]
Hi, I'm asking this question one and a half year ago https://github.com/stratis-storage/stratisd/issues/2570.

I've love to ask again, are we having any sort of built in data protection yet (or atleast on roadmap)?

Right now I'm using linux mdadm raid and still waiting for stratis to production ready on that aspect yet :heart:

**Comment by jbaublitz** [comment, marked as answer]
Please see #503 for the discussion that we'll be having around the roadmap and various tradeoffs.
